web3.go

```diff
@@ -160,23 +164,64 @@ func CallTransactFunction(ctx context.Context, client Client, myabi abi.ABI, add
 	}
 	return convertTx(signedTx, fromAddress), nil
 }
+func isValidUrl(toTest string) bool {
+	_, err := url.ParseRequestURI(toTest)
```

Comment:
Does this work? Seems too strict from the docs. Either way, shouldn't one version of parse should be enough?

Reply:
removed unnecessary check